protobuf/lib/src/protobuf/generated_message.dart

```diff
@@ -61,6 +61,11 @@ abstract class GeneratedMessage {
     return this;
   }
 
+  /// Returns `true` if this message is marked read-only. Otherwise `false`.
+  ///
+  /// Even when false, some sub-message could be read-only.
```

Comment:
when true, can submessages be modifiable?

Reply:
Should not happen, we always freeze submessages recursively. Clarified.